mm2src/mm2_main/src/mm2_tests.rs

```diff
@@ -6789,6 +6789,45 @@ fn test_mm2_db_migration() {
     .unwrap();
 }
 
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn test_get_current_mtp() {
+    use mm2_test_helpers::for_tests::Mm2TestConf;
+
+    // KMD coin config used for this test
+    let coins = json!([
+        {"coin":"KMD","txversion":4,"overwintered":1,"txfee":10000,"protocol":{"type":"UTXO"}},
+    ]);
+    let passphrase = "cMhHM3PMpMrChygR4bLF7QsTdenhWpFrrmf2UezBG3eeFsz41rtL";
+
+    let conf = Mm2TestConf::seednode(&passphrase, &coins);
```

Comment:
I think it's overhead to run a seednode in this test.
Seednode opens a server socket and listens for incoming P2P connections.
But anyway it's not critical now